pkg/cmd/server/start/master_args.go

```diff
@@ -348,7 +348,7 @@ func (args MasterArgs) BuildSerializeableOAuthConfig() (*configapi.OAuthConfig,
 
 		IdentityProviders: []configapi.IdentityProvider{},
 		GrantConfig: configapi.GrantConfig{
-			Method: "auto",
+			Method: configapi.GrantHandlerAuto,
```

Comment:
the intention is to remove the "openshift start" command in (near) future... i think having this here now is fine, but I would not focus on this command :)